packages/cli/src/platform/login.ts

```diff
@@ -27,6 +27,13 @@ export class Login implements Command {
   }
 
   public async parse() {
+    const authConfig = await readAuthConfig()
+    if (isError(authConfig)) throw authConfig
+    if (authConfig.token)
+      return `Already authenticated. Run ${green(
+        '`prisma platform auth show --early-access`',
+      )} to see the current user.`
```

Comment:
```suggestion
      return `Already authenticated. Run ${green(
        getCommandWithExecutor('prisma platform auth show --early-access'),
      )} to see the current user.`
```
Use `getCommandWithExecutor()` for displaying commands

Reply:
and add the import